prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -105,72 +98,41 @@ func handle(ghc githubClient, oc ownersClient, log *logrus.Entry, reviewerCount
 	return nil
 }
 
-func chance(val, total int64) float64 {
-	return 100.0 * float64(val) / float64(total)
-}
-
-func getPotentialReviewers(owners ownersClient, author string, files []github.PullRequestChange, leafOnly bool) (weightMap, int64) {
-	potentialReviewers := weightMap{}
-	weightSum := int64(0)
-	var fileOwners sets.String
+func getReviewers(owners ownersClient, author string, files []github.PullRequestChange, minReviewers int) ([]string, error) {
+	authorSet := sets.NewString(author)
+	reviewers := sets.NewString()
+	leafReviewers := sets.NewString()
+	// first build 'reviewers' such that each file has at least one leaf reviewer in the set.
 	for _, file := range files {
-		fileWeight := int64(1)
-		if file.Changes != 0 {
-			fileWeight = int64(file.Changes)
-		}
-		// Judge file size on a log scale-- effectively this
-		// makes three buckets, we shouldn't have many 10k+
-		// line changes.
-		fileWeight = int64(math.Log10(float64(fileWeight))) + 1
-		if leafOnly {
-			fileOwners = owners.LeafReviewers(file.Filename)
-		} else {
-			fileOwners = owners.Reviewers(file.Filename)
+		fileLeaf := owners.LeafReviewers(file.Filename).Difference(authorSet)
```

Comment:
Is this every changed file or just the OWNERS files? Seems like we first want to get the list of OWNERS files and then choose one reviewer from each OWNERS file